### PR TITLE
436 tests for tusd

### PR DESCRIPTION
**Pull request by alexkiro**
Adds some basic tests for the backend hooks for TUSD.

Only two simple test cases added for success scenario party/secretariat. TODOs added for some edge cases.

### CODE REVIEW

File: ozone/core/api/views.py [hunk 1]
```diff
@@ -886,7 +886,7 @@ def handle_post_finish(request):
             file_info_path.unlink()
 
         except UploadToken.DoesNotExist:
-            log.error(f'UPLOAD denied for "{token.user}": INVALID TOKEN')
+            log.error(f'UPLOAD denied for "{tok}": INVALID TOKEN')
```
Comment (sorinstelian) [inline ozone/core/api/views.py]: :+1:
